exploitation/enumeration/suspicious-ntquerysysteminformation-call.yml

```diff
@@ -0,0 +1,23 @@
+rule:
+  meta:
+    name: suspicious NtQuerySystemInformation call
```

Comment:
Let's update this name to more closely follow our naming guidelines: https://github.com/mandiant/capa-rules/blob/master/doc/format.md#rule-name